creator-node/src/diskManager.ts

```diff
@@ -757,31 +744,50 @@ const _migrateFilesWithCustomStoragePaths = async (
         newResultsFound = !isEqual(prevResults, results)
         // Process sequentially to minimize load since this is not time-sensitive
         let numFilesMigratedSuccessfully = 0
-        let numFilesFailedToMigrate = 0
+        const errorsWithoutPaths: string[] = []
         for (const fileRecord of results) {
-          let success
+          let success, error, newPath
           try {
-            success = await _migrateFileWithCustomStoragePath(
-              fileRecord,
-              logger
-            )
+            ;({ success, error, newPath } =
+              await _migrateFileWithCustomStoragePath(fileRecord, logger))
           } catch (e: any) {
             logger.error(
               `Error fixing fileRecord ${JSON.stringify(fileRecord)}: ${e}`
             )
-            numFilesFailedToMigrate++
+            success = false
+            error = e
+          }
+          if (success) {
+            numFilesMigratedSuccessfully++
+          } else {
+            const errWithoutPaths = replaceAll(
+              replaceAll(
```

Comment:
i like the cleverness here but same point as above, complexity > utility?

Reply:
similar to the other one, I'm not loving this way of visibility but without it I'm worried we won't be able to debug what remaining files aren't migrated on SPs, especially if they don't export logs or aren't in debug mode. easy to remove if it is an issue tho